### ISSUE TITLE
conda create can't create an env on Windows

### ISSUE THREAD

**Issue by wdwind** [conda/conda#1090]
Firstly create a conda env by using -p:

```
conda create -p d:/env3 python=2.7
```

Then check all envs by:

```
conda info -e
```

The result:

```
# conda environments:
#
env2                     C:\Anaconda\envs\env2
root                  *  C:\Anaconda
```

**No env3**.

And also `activate d:/env3` doesn't work.

**Comment by asmeurer** [comment]
conda info -e doesn't list envs created with -p and currently activate on windows doesn't support them. But the environment should be created correctly. You'll have to use absolute paths to access the Python.

**Comment by ovz** [comment]
Could we please consider adding support for absolute paths to activate.bat? When scripting a dev environment for a particular project it is a major inconvenience having to create environment for a project in c:\Anaconda\envs instead of project's build directory.

**Comment by kalefranz** [comment]
> Could we please consider adding support for absolute paths to activate.bat?

Included as part of #5044 and follow-on PRs for conda 4.4, which will be rolling through the canary process over the next month.
